### DISCUSSION TITLE
[Docs] Add styles to the Examples

### DISCUSSION THREAD

**Discussion by mmellado** [chakra-ui/zag#733] [Ideas]
# 🚀 Feature request

Add tab with the example's code to the working example at the top of every component's documentation page.

## 🧱 Problem Statement / Justification

Today I spent about 30min trying to figure out why my menu was not disappearing when I was clicking outside of it or clicking on one of its items... only to find out it was because I was meant to handle this using CSS. This is fine, and it's part of what I like about the UI agnostic aspects of Zag, but I think it's something that's easy for new users to forget. 

We have a code example at the bottom of the page for each framework, but these do not include styles, only the functional code, so maybe, if the source to the code of the working example at the top was present, it might make it easier for people to understand all the important parts of styling, as I felt the styleguide at the bottom was not enough (maybe I'm just a very visual person)

**Comment by anubra266** [comment]
@mmellado there are styles for all components here: https://github.com/chakra-ui/zag/blob/main/shared/src/style.css

**Comment by mmellado** [comment]
Yes. I’m aware of that but for documentation purposes, easy access to them
directly from the docs page would be very useful, specially to new users.

**Comment by segunadebayo** [comment]
Closing as outdated — the documentation and website have been fully rebuilt for v1.0.
